autoprotocol/protocol.py

```diff
@@ -1030,9 +1031,15 @@ def liquid_handle_dispense(
         columns: int = 1,
         method: DispenseMethod = DispenseMethod,
         liquid: LiquidClass = LiquidClass,
+        device: Optional[str] = "x_tempest_chip",
         model: Optional[str] = None,
         chip_material: Optional[str] = None,
         nozzle: Optional[str] = None,
+        diaphragm: Optional[int] = None,
+        nozzle_size: Optional[Unit] = None,
```

Comment:
technically we accept Optional Unit or str 🤔

Reply:
AGH ok yeah I think I resolved it? The output will always spit out a `Unit` type _but_ accepts both string and unit. I'll do the same for `z_drop`